addon/-private/system/many-array.js

```diff
@@ -136,7 +136,12 @@ export default Ember.Object.extend(Ember.MutableArray, Ember.Evented, {
   objectAt(index) {
     let object = this.currentState[index];
     //Ember observers such as 'firstObject', 'lastObject' might do out of bounds accesses
-    if (object === undefined) { return; }
+    if (object === undefined) {
```

Comment:
for future travelers the issue @runspired is referring to is https://github.com/emberjs/ember.js/issues/14843